### PR TITLE
Apartment Cards

### PR DESCRIPTION
**Pull request by cyrus14**
### Summary 

This pull request works towards completing #150 by implementing the Apartment Card listings on the homepage. Some changes to the formatting have to be done still, but the main properties are present.
- Changed from landowner name to Apartment name
- Switched Graphic to be on side of the Card rather than the top
- Only 1 card per row

### Test Plan 
Navigate to the root directory and run yarn start
Apartment listings should match the corresponding components on the Figma. This includes the placeholder graphic, name, address, reviews, and review text.

https://user-images.githubusercontent.com/60114462/161145449-628f011d-2fb1-48e2-9758-cac6e24b99a1.mov

### CODE REVIEW

File: frontend/src/components/ApartmentCard/ApartmentCard.tsx
Comment (samliu10) [inline frontend/src/components/ApartmentCard/ApartmentCard.tsx:42]: Try experimenting with `borderRadius` directly on the image - right now it shows up on the `Card` container but sometimes not on the image itself after you add padding so that there's space between the image and the containing `Card` component (add padding to the `<Grid item>` component containing `CardMedia`

File: frontend/src/components/ApartmentCard/ApartmentCard.tsx
Comment (samliu10) [inline frontend/src/components/ApartmentCard/ApartmentCard.tsx:61]: What if we take `HeartRating` out of `Typography`? It'll probably be easier to style them if they're separate. If you add `alignItems` to the parent container, it should vertically align the text with the hearts. 
```
  <Grid container direction="row" alignItems="center">
    <HeartRating value={3} readOnly />
    <Typography variant="h6" className={classes.reviewNum}>
      {numReviews + (numReviews !== 1 ? ' Reviews' : ' Review')}
    </Typography>
  </Grid>
```

Also, you could try adding a `marginLeft` to the styling on the text so that it's not so close to the hearts, kinda like this: 
Screen Shot 2022-03-20 at 5 45 18 PM

File: frontend/src/components/ApartmentCard/ApartmentCard.tsx
Comment (samliu10) [inline frontend/src/components/ApartmentCard/ApartmentCard.tsx:2]: Great job!! The last thing is figuring out how to make the cards not get too big vertically, i.e. restraining the height of the cards while responsively sizing the width. Tbh my frontend/css knowledge isn't that extensive, so we can play around with it and see what we can do

**Review by samliu10 — CHANGES_REQUESTED** [review]

File: frontend/src/components/ApartmentCard/ApartmentCard.tsx
Comment (samliu10) [inline frontend/src/components/ApartmentCard/ApartmentCard.tsx:15]: Try not to use absolute sizing — if you resize the page right now, you'll find that the words spill over onto the image. If you change this to percentages, it should preserve responsive sizing!

**Comment by thuypham03** [comment]
It's amazing that you've self-learned MUI and made significant progress on your task just in a week! However, the heart rating section and the number of reviews is not in a row, maybe you can try to change margin top/bottom of the heart section to adapt them.

File: frontend/src/components/ApartmentCard/ApartmentCard.tsx
Comment (samliu10) [inline frontend/src/components/ApartmentCard/ApartmentCard.tsx:27]: Don't forget to delete`msgImg` and `msg.png` since they're not used. Otherwise, great job!!!! Wait for Jiayuan to merge #168 first before merging this PR.

**Review by samliu10 — APPROVED** [review]

File: frontend/src/components/ApartmentCard/ApartmentCard.tsx
Comment (cyrus14) [reply frontend/src/components/ApartmentCard/ApartmentCard.tsx:27]: Doing rn, thanks! I will wait for Jiayuan.

**Review by samliu10 — APPROVED** [review]

**Pull request merged**
